ckanext/stadtzhtheme/templates/package/snippets/package_basic_fields.html

```diff
@@ -11,7 +11,7 @@
   <label class="form-label" for="field-license">{{ _("License") }}</label>
   <div class="controls">
     <select id="field-license" name="license_id" data-module="autocomplete">
-      {% for license_desc, license_id in licenses|sort if license_desc  %}
+      {% for license_id, license_desc in h.license_options(existing_license_id)  %}
```

Comment:
maybe it is defined in an upper template, or is `existing_license_id` actually not defined here?

Reply:
thanks for spotting that! i thought it was defined in an upper template but it should be defined here. :sweat_smile: